### PR TITLE
[modular] [ready] the ever illusive holobadge port

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

its here, I did it

## Why It's Good For The Game

you can turn in your badge and gun when you get fired for beating the clown, or doing lewd things with the prisoners
you can swish it around and tell DIS IS DA POLICE
also I added a light to the detective's office on tram

## Changelog
:cl:
add: Holobadges for...the majority of sec
fix: added a missing light to the det office in tram
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+		to_chat(user, "Waving around a holobadge before swiping an ID would be pretty pointless.")
+		return
+	return ..()
+
+/obj/item/clothing/accessory/badge/holo/emag_act(var/remaining_charges, var/mob/user)
+	if (emagged)
+		to_chat(user, "<span class='danger'>\The [src] is already cracked.</span>")
+		return
+	else
+		emagged = 1
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/accessories.dm]:
```suggestion
		emagged = TRUE
```

File: modular_skyrat/modules/customization/modules/clothing/under/accessories.dm [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+		to_chat(user, "<span class='danger'>You crack the holobadge security checks.</span>")
+		return 1
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/accessories.dm]:
```suggestion
		return TRUE
```

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/customization/__DEFINES/mobs.dm [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#define DEFAULT_QUICK_COOLDOWN  4
```
Comment (Funce) [inline modular_skyrat/modules/customization/__DEFINES/mobs.dm]: Unused define?

Comment (thestubborn) [reply modular_skyrat/modules/customization/__DEFINES/mobs.dm]: yes, actually, let me delete and remove that

**Pull request merged**